src/clj/puppetlabs/services/jruby_pool_manager/jruby_schemas.clj

```diff
@@ -59,14 +59,18 @@
         values are `:jit`, `:force`, and `:off`.  Defaults to `:off`.
 
     * :max-active-instances - The maximum number of JRubyInstances that
-        will be pooled."
+        will be pooled.
+
+    * :environment-vars - A map of whitelisted environment variables and their values to be
+        whitelisted and visible to any Ruby code."
   {:ruby-load-path [schema/Str]
    :gem-home schema/Str
    :compile-mode SupportedJRubyCompileModes
    :borrow-timeout schema/Int
    :max-active-instances schema/Int
    :max-borrows-per-instance schema/Int
-   :lifecycle LifecycleFns})
+   :lifecycle LifecycleFns
+   :environment-vars {schema/Str schema/Str}})
```

Comment:
One thing to test out... I think that when the TK config service reads in HOCON files it usually makes keys in maps into clojure keywords. If that's the case, should this be keyword? Or would we require users to put things in the hocon file in double quotes (I think - although I'm not sure - that this would make them strings). It seems like it would be good to do a little manual testing of this.